test/gtest/uct/test_p2p_err.cc

```diff
@@ -243,6 +243,10 @@ UCS_TEST_P(uct_p2p_err_test, invalid_am_bcopy_length) {
 }
 
 UCS_TEST_P(uct_p2p_err_test, invalid_am_zcopy_hdr_length) {
+    if (GetParam()->tl_name == "ud") {
+        UCS_TEST_SKIP_R("UD has no limit for zcopy hdr length");
```

Comment:
why need to skip it?

Reply:
it failed because UD is able to send header longer than max_inline

Reply:
can we update test logic to avoid skip? at least if max_inline is big enough?

Reply:
updated... but it required to add more complex evaluation in ud_am_zcopy call